Open Judge System/Data/OJS.Data/Migrations/DefaultMigrationConfiguration.cs

```diff
@@ -357,7 +357,7 @@ protected void SeedSubmissionTypes(OjsDbContext context)
                 {
                     Name = "C++ Zip File",
                     CompilerType = CompilerType.CPlusPlusZip,
-                    AdditionalCompilerArguments = "-pipe -mtune=generic -O3 -static-libgcc -static-libstdc++",
```

Comment:
We decided optimization is not required at all due to the fact that the results of it are negligible compared to the possible performance hit.